src/managers/channel-manager.js

```diff
@@ -3,18 +3,35 @@
 var Channel = require('../service/channel-service');
 
 /**
- * Generic wrapper around $.cometd. This provides a few nice features the default cometd wrapper doesn't
+ * ## Channel Manager
  *
- * - Automatic re-subscription to channels if you lose your connection in-between
- * - Online/ Offline notifications
- * - 'Events' for cometd notifications, instead of having to listen on specific meta channels
+ * The Epicenter platform provides a push channel, which allows you to publish and subscribe to messages within a [project](../../../glossary/#projects), [group](../../../glossary/#groups), or [multiplayer world](../../../glossary/#world). There are two main use cases for the channel: event notifications and chat messages.
```

Comment:
I'd rather leave out the Epicenter references from here, and just have them in the epicenter-channel-manager. Is there a reason to have this here?